UI/data/locale/en-US.ini

```diff
@@ -257,6 +257,10 @@ ConfirmStart.Text="Are you sure you want to start the stream?"
 ConfirmStop.Title="Stop Stream?"
 ConfirmStop.Text="Are you sure you want to stop the stream?"
 
+# confirm bandwidth test dialog box
+ConfirmBWTest.Title="Start Bandwidth Test?"
+ConfirmBWTest.Text="You have OBS configured in bandwidth test mode, this mode allows for network testing without your channel going live. You will have to disable it if you want to start streaming.\nDo you want to continue?"
```

Comment:
Small nit: `...bandwidth test mode. This mode allows...`

Also it might look a bit better with `\n\n` before `Do you want to continue?` but I could go either way.

Reply:
Also I'm not sure about `if you want to start streaming`. This will let them start streaming, but no one will be able to see their stream. I don't want to imply that in bandwidth test mode they can't do a test stream. Not sure on a better phrasing though. Maybe `Once you are done testing, you will need to disable it so that you can go live.`

Reply:
I have considered altering the phrasing to something like `you will need to disable it in order for your viewers to be able to watch your stream`

Reply:
That would do it, too.